observability/opentelemetry/src/main/java/org/wildfly/extension/opentelemetry/OpenTelemetryParser_1_0.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ *
+ * Copyright 2021 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wildfly.extension.opentelemetry;
+
+import static org.jboss.as.controller.PersistentResourceXMLDescription.builder;
+
+import org.jboss.as.controller.PersistentResourceXMLDescription;
+import org.jboss.as.controller.PersistentResourceXMLParser;
+
+public class OpenTelemetryParser_1_0 extends PersistentResourceXMLParser {
```

Comment:
I suggest parameterize your parser with a schema version, rather than anticipate a parser class per version.  You'll save yourself a lot of code duplication in the long run.
e.g. https://github.com/wildfly/wildfly/blob/main/clustering/web/extension/src/main/java/org/wildfly/extension/clustering/web/DistributableWebXMLParser.java

Reply:
Done.